backend/app/routes/admin/inviteAdmin.js

```diff
@@ -0,0 +1,21 @@
+const to = require('await-to-js').default;
+const { ErrorHandler } = require('../../../helpers/error');
+const { sendEmail } = require('../../../helpers/emailService');
+const constants = require('../../../constants');
+
+module.exports = async (req, res, next) => {
+  const { email, data } = req.body;
+  const type = 'INVITE-ADMIN';
+  const [err] = await to(sendEmail(email, data, type));
```

Comment:
Ideally we want to incorporate feature for sending emails to multiple people, can you incorporate that?
@Kajol-Kumari what do you suggest? Do we keep it for a single admin or multiple admins?

Reply:
should I implement the multiple people feature?

Reply:
Hey @priyanshu20 please keep it for single admin invite as we would be sending the invite done by one only
\cc - @jackfrost13